product/tutorials/tutorial_set_inventory.md

```diff
@@ -0,0 +1,105 @@
+# **Set Inventory**
+
+## Let's get started
+
+Changes to a product's inventory information may occur much more frequently than changes to its catalog information.
+
+You may decide to issue inventory updates instead of updating entire product.
```

Comment:
You may decide to push inventory updates instead of updating the entire product.